.gitignore

```diff
@@ -68,3 +67,36 @@ iOS/nativelibs/arm64
 iOS/nativelibs/i386
 iOS/nativelibs/x86_64
 Solutions/Engine.App/test.ini
+
+libsrc/*.tar*
+libsrc/*.zip
+
+ac2game.dat
+audio.vox
+speech.vox
```

Comment:
Ahh, that makes sense.  And correct, Wadjet Eye still use "ac2game.dat" for their current games.  They might just be following the convention set by @jjsat though.